docs/sources/reference/commandline/cli.md

```diff
@@ -17,6 +17,31 @@ or execute `docker help`:
 
       ...
 
+## Environment Variables
+
+For easy reference, the follow list of environment variables will be used by
+the `docker` command line:
+
+* `DOCKER_CERT_PATH` The location of your authentication keys.
+* `DOCKER_DRIVER` The graph driver to use.
+* `DOCKER_HOST` Daemon socket to connect to.
+* `DOCKER_NOWARN_KERNEL_VERSION` If set docker will not print a warning about your Linux kernel being unsuitable for Docker.
+* `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
+* `DOCKER_TLS_VERIFY` If set this will use TLS and verify the remote.
+* `DOCKER_TMPDIR` Location for temporary Docker files.
+
+Due to Docker being developed using 'Go', any environment variables
+used by the 'Go' runtime may also be used. In particular the following
+might be of interest:
+
+* `HTTP_PROXY`
+* `HTTPS_PROXY`
+* `NO_PROXY`
+
+These can be specified in either upper or lower case. See the
```

Comment:
_You can specify these in upper or lower case._

Or do you mean "Go environment variables are case-insensitive"  or is it "HTTP header names are case-insensitive."